csss/utilities.py

```diff
@@ -0,0 +1,184 @@
+from importlib import reload
+import datetime as dt
+import numpy as np
+from numpy.linalg import svd, pinv
+from scipy.stats import ttest_ind
+import sqlalchemy as sq
+import pandas as pd
+import json
+from time import time
+from os import path
+import os
+from pathlib import Path
+import pickle as pk
+import matplotlib.pyplot as plt
+from csss.SolarDisagg import createTempInput
+from sklearn.linear_model import LinearRegression
+from sklearn.preprocessing import StandardScaler
+import Custom_Functions.error_functions as ef
+from IPython import get_ipython
+
+def load_setup():
+
+    START_DATE = '01-01-2015'
```

Comment:
Please change from hard-coded dates to function args or kwargs. Also, all caps variables are reserved for globals, so please use lower case for internal function variables.